packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -0,0 +1,215 @@
+              @slot History Query content
+                  @binding {Suggestion} suggestion - History Query suggestion data
+                  @binding {string} queryHTML - Suggestion's query with the matching part inside a span tag
+                  @binding {number} index - History Query suggestion index
+            -->
+            <!-- eslint-enable max-len -->
+            <slot name="suggestion-content" v-bind="{ suggestion, index, queryHTML }" />
+          </template>
+          <template #remove-button-content>
+            <!--
+              @slot History Query remove button content
+                  @binding {Suggestion} suggestion - History Query suggestion data
+                  @binding {number} index - History Query suggestion index
+            -->
+            <slot name="suggestion-remove-content" v-bind="{ suggestion, index }" />
+          </template>
+        </HistoryQuery>
+      </slot>
+    </template>
+  </BaseSuggestions>
+</template>
+
+<script lang="ts">
+  import { HistoryQuery } from '@empathyco/x-types';
+  import Vue from 'vue';
+  import { Component, Prop } from 'vue-property-decorator';
+  import { State, xComponentMixin } from '../../../components';
+  import BaseSuggestions from '../../../components/suggestions/base-suggestions.vue';
+  import { historyQueriesXModule } from '../x-module';
+  import HistoryQueryComponent from './history-query.vue';
+
+  /**
+   * This component renders the complete list of suggestions coming from the user queries history.
+   *
+   * @remarks
+   *
+   * Allows the user to select one of them, emitting the needed events.
+   * A history query is just another type of suggestion that contains a query that the user has
+   * made in the past.
+   *
+   * @public
+   */
+  @Component({
+    components: { HistoryQuery: HistoryQueryComponent, BaseSuggestions },
+    mixins: [xComponentMixin(historyQueriesXModule)]
+  })
+  export default class MyHistory extends Vue {
+    /**
+     * Animation component that will be used to animate the suggestions.
+     *
+     * @public
+     */
+    @Prop()
+    protected animation!: Vue;
+
+    /**
+     * The list of history queries.
+     *
+     * @internal
+     */
+    @State('historyQueries', 'historyQueries')
+    public historyQueries!: HistoryQuery[];
+  }
+</script>
+
+<docs lang="mdx">
+## Events
+
+This component doesn't emit events.
+
+## See it in action
+
+Here you have a basic example of how the MyHistory is rendered.
+
+```vue
+<template>
+  <MyHistory />
+</template>
+
+<script>
+  import { MyHistory } from '@empathyco/x-components/history-queries';
+
+  export default {
+    name: 'MyHistoryDemo',
+    components: {
+      MyHistory
+    }
+  };
+</script>
+```
+
+### Play with the animation
+
+```vue
+<template>
+  <MyHistory :animation="fadeAndSlide" />
+</template>
+
+<script>
+  import { MyHistory } from '@empathyco/x-components/history-queries';
+  import { FadeAndSlide } from '@empathyco/x-components/animations';
```

Comment:
There is no `animations` entry point, `FadeAndSlide` should be imported from `@empathyco/x-components`